src/Components/CreateImageWizardV2/steps/ImageOutput/ReleaseLifecycle.tsx

```diff
@@ -103,7 +103,7 @@ export const chartMajorVersionCfg = {
   },
 };
 
-const MajorReleasesLifecyclesChart = () => {
+export const MajorReleasesLifecyclesChart = () => {
```

Comment:
Does this change belong to this PR?

Reply:
no, do you think I need to open new pr for that?